teachopencadd/talktorials/T011_query_online_api_webservices/talktorial.ipynb

```diff
@@ -4,2811 +4,1348 @@
    "cell_type": "markdown",
```

Comment:
<p>missing word: SARS-CoV-2 is one <strong>of</strong> the most popular</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>